class-workflow.php

```diff
@@ -120,6 +121,8 @@ private function load_modules() {
 	private function setup_actions() {
 		add_action( 'init', [ $this, 'action_init' ] );
 		add_action( 'init', [ $this, 'action_init_after' ], 1000 );
+		// Load the telemetry module in init as all plugins are loaded by then
+		add_action( 'init', [ $this, 'start_tracking' ] );
```

Comment:
Maybe `init_telemetry` as a function name alternative?

Reply:
Can this function be called directly in the telemetry module's `init()`? It would seem more relevant in there.